Xamarin.Essentials/SecureStorage/SecureStorage.shared.cs

```diff
@@ -10,7 +10,7 @@ public static partial class SecureStorage
 
         // Special alias needed to used grouping
         internal static string GetAlias(string accessGroup)
-           => string.IsNullOrWhiteSpace(accessGroup) ? Alias : $"{accessGroup}.{Alias}";
+           => string.IsNullOrWhiteSpace(accessGroup) ? Alias : accessGroup;
```

Comment:
This one concerns me as this is shared so would break all other platforms. We should not change this.

Reply:
Thanks for reviewing!  Without this on iOS, shared, secure access won't work, so it sounds like this aliasing mechanism will have to be a bit more platform specific.  Will try to come up with something different and resubmit as soon as I can.  Thanks again!

Reply:
I mean in that case I would just change your logic not to call this method and just call this locally in each method:

```
string.IsNullOrWhiteSpace(accessGroup) ? Alias : accessGroup;
```

Reply:
checkout my commit

Reply:
nm this is good, saw other comments